sdk/lusid/utilities/api_client_builder.py

```diff
@@ -85,6 +87,14 @@ def build(cls, api_secrets_filename=None, id_provider_response_handler=None, api
         config.access_token = api_token
```

Comment:
Why don't you pass the value in like this `config = Configuration(tcp_keep_alive=tcp_keep_alive, flag_to_use_decimal=flag_to_use_decimal)`?